wireapp/wire-android
# fix: fixed ping translations in German [WPB-4728]

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-4728" title="WPB-4728" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />WPB-4728</a>  Applause -[Android] Chat - Poor translated ping message
  </td></table>
  <br />
 






----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Knock system messages translations were still not discriminating between self and other users triggering the action, hence the German translation was broken (`Sie hat gepingt` vs `Sie haben gepingt`)

### Solutions

Added a flag to discern whether self user triggered the action or not.

### Testing

#### Test Coverage (Optional)


#### How to Test

- Change locale to German.
- Go into a conversation.
- Send a ping.
- Receive a ping.
- Check translations are correct

### Attachments (Optional)


| Before | After |
| ----------- | ------------ |
| ![Captura de pantalla 2023-09-27 a las 11 48 37](https://github.com/wireapp/wire-android-reloaded/assets/2468164/3ff1a384-82d5-48fc-9a21-e38a09b023b4)|   ![Captura de pantalla 2023-09-27 a las 11 46 33](https://github.com/wireapp/wire-android-reloaded/assets/2468164/1b726893-d701-41e3-9577-e4c7ecc97f63) |

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
